cirq/experiments/qubit_characterizations.py

```diff
@@ -0,0 +1,567 @@
+import itertools
+from typing import Sequence, Dict
+from cirq import circuits, devices, ops, protocols, sim, study, value
+import numpy
+from matplotlib import pyplot
+from mpl_toolkits.mplot3d import Axes3D
+
+
+class RabiExperimentResult(object):
+    """Class for storing and plotting data from a Rabi oscillation
+    experiment.
+    """
+
+    def __init__(self, num_shots: int, rabi_angles: Sequence[float],
+                 excited_state_pops: Sequence[float]):
```

Comment:
rename `pops` to `probs`

Reply:
All "...pops" have been renamed to "...probs"